docs/remote.go

```diff
@@ -27,61 +27,61 @@ const (
-	that interacts with singularity services.`
+The 'remote use' command sets the remote to be used by default by any command
+that interacts with singularity services.`
 	RemoteUseExample string = `
-	$ singularity remote use SylabsCloud`
+$ singularity remote use SylabsCloud`
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	// remote list command
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	RemoteListUse   string = `list`
 	RemoteListShort string = `List all singularity remote endpoints that are configured`
 	RemoteListLong  string = `
-	The 'remote list' command lists all remote endpoints configured for use. If you 
-	have set a remote as a default, its name will be encompassed by brackets.`
+The 'remote list' command lists all remote endpoints configured for use. If you 
+have set a remote as a default, its name will be encompassed by brackets.`
 	RemoteListExample string = `
-	$ singularity remote list`
+$ singularity remote list`
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	// remote login command
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	RemoteLoginUse   string = `login <remote_name>`
 	RemoteLoginShort string = `Log into a singularity remote endpoint using an authentication token`
 	RemoteLoginLong  string = `
-	The 'remote login' command allows you to set an authentication token for a
-	specific endpoint. This command will produce a link directing you to the token
-	service you can use to generate a valid token. If no endpoint is specified,
-	it will try the default remote.`
+The 'remote login' command allows you to set an authentication token for a
+specific endpoint. This command will produce a link directing you to the token
+service you can use to generate a valid token. If no endpoint is specified,
+it will try the default remote.`
 	RemoteLoginExample string = `
-	$ singularity remote login SylabsCloud`
+$ singularity remote login SylabsCloud`
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	// remote status command
 	// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 	RemoteStatusUse   string = `status <remote_name>`
 	RemoteStatusShort string = `Check the status of the singularity services at an endpoint`
 	RemoteStatusLong  string = `
-	the 'remote status' command checks the status of the specified remote endpoint
-	and reports the availibility of services and their versions. If no endpoint is
-	specified, it will check the status of the default remote.`
+The 'remote status' command checks the status of the specified remote endpoint
+and reports the availibility of services and their versions. If no endpoint is
+specified, it will check the status of the default remote.`
 	RemoteStatusExample string = `
-	$ singularity remote status SylabsCloud`
+$ singularity remote status SylabsCloud`
```

Comment:
Here too, for this whole file

```suggestion
  $ singularity remote status SylabsCloud`
```